app/controllers/course/assessment/question/controller.rb

```diff
@@ -29,6 +32,18 @@ def authorize_create_question_in_koditsu
     raise CanCan::AccessDenied if @assessment.is_koditsu_enabled && is_course_koditsu_enabled
   end
 
+  def flag_question_not_synced_with_koditsu
```

Comment:
Since this method is in already in `question/controller` you can align the naming to `flag_not_synced_with_koditsu` instead of `flag_question_not_synced_with_koditsu`.

Reply:
done